include/fmt/chrono.h

```diff
@@ -764,13 +764,20 @@ inline std::chrono::duration<Rep, std::milli> get_milliseconds(
 
 template <typename Char, typename Rep, typename OutputIt>
 OutputIt format_duration_value(OutputIt out, Rep val, int precision) {
+  using context = FMT_BUFFER_CONTEXT(type_identity_t<Char>);
+
   const Char pr_f[] = {'{', ':', '.', '{', '}', 'f', '}', 0};
-  if (precision >= 0) return format_to(out, pr_f, val, precision);
+  if (precision >= 0) {
+    return vformat_to(out, to_string_view(pr_f),
+                      make_format_args<context>(val, precision));
+  }
```

Comment:
Please make `pr_f``static constexpr` and keep `format_to` (don't change it to `vformat_to`). `static constexpr` strings should work with `FMT_STRING`:

```c++
static constexpr const Char pr_f[] = {'{', ':', '.', '{', '}', 'f', '}', 0};
  if (precision >= 0) return format_to(out, FMT_STRING(pr_f), val, precision);
```

Same below.

Reply:
ah, good to know! Won't that break on platforms where FMT_CONSTEXPR_DECL evaluates to noop though?

Reply:
On that platform `FMT_STRING` should also evaluate to noop (at least in theory).

Reply:
Nm, tested on godbolt and realized that's a perfectly valid c++11 feature

Reply:
This apparently breaks on windows, even after correcting it to use SFINAE to avoid compiling the `{:.{}f}` format string with an integral type. I'm adding a test for using FMT_STRING with a static FMT_CONSTEXPR_DECL, and I'll see if I can get a workaround